### PR TITLE
First Journey:NUA

### PR DESCRIPTION
**Pull request by GeronimoJohn**
- Update seed to follow the NUA first journey: Fact or Fiction in [figma](https://www.figma.com/file/Rt5hcy8zJIUwz4SmnfniSG/NextSteps---Design-System?node-id=432%3A5453)

### CODE REVIEW

File: apps/api-journeys/prisma/factories/nua1.ts [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+      parentOrder: 0
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: card2.id,
+      extraAttrs: {
+        content: 'A QUICK QUESTION...',
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua1.ts:209]:
```suggestion
        content: 'What do you think?',
```

File: apps/api-journeys/prisma/factories/nua1.ts [hunk 2]
```diff
@@ -0,0 +1,590 @@
...
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: card4.id,
+      extraAttrs: {
+        label: 'One question remains...',
+        variant: 'contained',
+        color: 'primary',
+        size: 'medium',
+        startIcon: {
+          name: 'PlayArrow'
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua1.ts:411]: Should be contact support - can be done during cooldown.

File: apps/api-journeys/prisma/factories/nua1.ts [hunk 3]
```diff
@@ -0,0 +1,590 @@
...
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: gridItemLeft.id,
+      extraAttrs: {
+        content: "IF IT'S TRUE...",
+        variant: 'h6',
+        color: 'primary',
+        align: 'left'
+      },
+      parentOrder: 1
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua1.ts:502]:
```suggestion
      parentOrder: 0
```

File: apps/api-journeys/prisma/factories/nua1.ts [hunk 4]
```diff
@@ -0,0 +1,590 @@
...
+      blockType: 'RadioOptionBlock',
+      parentBlockId: question4.id,
+      extraAttrs: {
+        label: 'A fake historical figure',
+        action: {
+          gtmEventName: 'click',
+          journeyId: resurrection?.id
+        }
+      },
+      parentOrder: 3
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua1.ts:587]:
```suggestion
      parentOrder: 4
```

File: apps/api-journeys/prisma/factories/nua2.ts [hunk 5]
```diff
@@ -0,0 +1,583 @@
...
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: card.id,
+      extraAttrs: {
+        content:
+          "Jesus' was found empty three days after his death-what could have happened to the body?",
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua2.ts:88]:
```suggestion
          "Jesus’ tomb was found empty three days after his death—what could have happened to the body?",
```

File: apps/api-journeys/prisma/factories/nua2.ts [hunk 6]
```diff
@@ -0,0 +1,583 @@
...
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: card4.id,
+      extraAttrs: {
+        label: 'What does it mean?',
+        variant: 'contained',
+        color: 'primary',
+        size: 'medium',
+        startIcon: {
+          name: 'PlayArrow'
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua2.ts:426]:
```suggestion
          name: 'ContactSupport'
```
When we have it added

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 7]
```diff
@@ -0,0 +1,1527 @@
...
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: prayerCard1.id,
+      extraAttrs: {
+        label: 'Next',
+        variant: 'contained',
+        color: 'primary',
+        size: 'small',
+        startIcon: {
+          name: 'PlayArrow'
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:377]:
```suggestion
          name: 'ChevronRight'
```

If it's available - else add to our IconList during cooldown

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 8]
```diff
@@ -0,0 +1,1527 @@
...
+        locked: false,
+        nextBlockId
+      },
+      parentOrder: 4
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:369]: Also we'd need to look into making a button that's not full width. - might do during cooldown

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 9]
```diff
@@ -0,0 +1,1527 @@
...
+      parentBlockId: prayerCard2.id,
+      extraAttrs: {
+        label: 'Next',
+        variant: 'contained',
+        color: 'primary',
+        size: 'small',
+        startIcon: {
+          name: 'PlayArrow'
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:470]:
```suggestion
          name: 'ChevronRight'
```
When it's added into backend

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 10]
```diff
@@ -0,0 +1,1527 @@
...
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: prayerCard3.id,
+      extraAttrs: {
+        label: 'Amen',
+        variant: 'contained',
+        color: 'primary',
+        size: 'small',
+        startIcon: {
+          name: 'PlayArrow'
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:563]:
```suggestion
          name: 'BeenhereRound'
```

Need to add to Icons list

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 11]
```diff
@@ -0,0 +1,1527 @@
...
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'SignUpBlock',
+      parentBlockId: prayerGridItemRight.id,
+      extraAttrs: {
+        submitLabel: 'Submit',
+        submitIcon: {
+          name: 'PlayArrow'
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:1388]:
```suggestion
          name: 'SendRounded'
```

Add this icon

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 12]
```diff
@@ -0,0 +1,1527 @@
...
+      parentOrder: 1
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: card2.id,
+      extraAttrs: {
+        content: 'Ever though about telling a friend what this mean to you?',
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:734]:
```suggestion
        content: 'Ever thought about telling a friend what this means to you?',
```

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 13]
```diff
@@ -0,0 +1,1527 @@
...
+      parentOrder: 2
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: card2.id,
+      extraAttrs: {
+        content: 'Sharing your story helps you grow with God and others',
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:748]:
```suggestion
        content: 'Sharing your story helps you grow with God and others.',
```

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 14]
```diff
@@ -0,0 +1,1527 @@
...
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: card2.id,
+      extraAttrs: {
+        label: 'Share Now',
+        variant: 'contained',
+        color: 'primary',
+        size: 'large',
+        startIcon: {
+          name: 'PlayArrow'
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:796]:
```suggestion
          name: 'SendRounded'
```
When added

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 15]
```diff
@@ -0,0 +1,1527 @@
...
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: alreadyGridItemLeft.id,
+      extraAttrs: {
+        content: 'Get a few tips by email on how to share your faith',
+        variant: 'h4',
+        color: 'primary',
+        align: 'left'
+      },
+      parentOrder: 0
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:901]:
```suggestion
      parentOrder: 1
```

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 16]
```diff
@@ -0,0 +1,1527 @@
...
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: lastCard.id,
+      extraAttrs: {
+        label: 'Watch Other Videos',
+        variant: 'contained',
+        color: 'primary',
+        size: 'medium',
+        startIcon: {
+          name: 'PlayArrow'
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:1517]:
```suggestion
          name: 'SubscriptionsRounded'
```
Add during cooldown

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 17]
```diff
@@ -0,0 +1,1527 @@
...
+      parentOrder: 2
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: noThanksCard.id,
+      extraAttrs: {
+        content: "Don't stop here. Keep exploring an dasking questions.",
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:980]:
```suggestion
        content: "Don't stop here. Keep exploring and asking questions.",
```

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 18]
```diff
@@ -0,0 +1,1527 @@
...
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: noThanksCard.id,
+      extraAttrs: {
+        label: "What's Next?",
+        variant: 'contained',
+        color: 'primary',
+        size: 'large',
+        startIcon: {
+          name: 'PlayArrow'
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:1028]:
```suggestion
          name: 'ContactSupport'
```

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 19]
```diff
@@ -0,0 +1,1527 @@
...
+      parentOrder: 0
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: noThanksGridItemLeft.id,
+      extraAttrs: {
+        content: 'Get new released videos by email',
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:1128]:
```suggestion
        content: 'Get new released videos by email.',
```

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 20]
```diff
@@ -0,0 +1,1527 @@
...
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: noThanksGridItemLeft.id,
+      extraAttrs: {
+        content: "WHAT'S NEXT?",
+        variant: 'h6',
+        color: 'primary',
+        align: 'left'
+      },
+      parentOrder: 0
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:1119]: This parent order might be wrong? it's rendering like:

image

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 21]
```diff
@@ -0,0 +1,1527 @@
...
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'SignUpBlock',
+      parentBlockId: noThanksGridItemRight.id,
+      extraAttrs: {
+        submitLabel: 'Submit',
+        submitIcon: {
+          name: 'PlayArrow'
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:1405]:
```suggestion
          name: 'SendRounded'
```

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 22]
```diff
@@ -0,0 +1,1527 @@
...
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'SignUpBlock',
+      parentBlockId: alreadyGridItemRight.id,
+      extraAttrs: {
+        submitLabel: 'Submit',
+        submitIcon: {
+          name: 'PlayArrow'
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:1422]:
```suggestion
          name: 'SendRounded'
```

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 23]
```diff
@@ -0,0 +1,1527 @@
...
+      parentOrder: 2
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: notSureCard.id,
+      extraAttrs: {
+        content: 'God can answer your questions and help overcome worries',
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:1212]:
```suggestion
        content: 'God can answer your questions and help overcome worries.',
```

File: apps/api-journeys/prisma/factories/nua9.ts [hunk 24]
```diff
@@ -0,0 +1,1527 @@
...
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: notSureCard.id,
+      extraAttrs: {
+        label: "What's Next?",
+        variant: 'contained',
+        color: 'primary',
+        size: 'large',
+        startIcon: {
+          name: 'PlayArrow'
```
Comment (storyworks) [inline apps/api-journeys/prisma/factories/nua9.ts:1260]:
```suggestion
          name: 'ContactSupport'
```

**Comment by csiyang** [comment]
Falling plates, no longer works :(

Screenshot 2021-11-05 173242

File: apps/api-journeys/prisma/seed.ts [hunk 25]
```diff
@@ -1,399 +1,14 @@
-import { PrismaClient, ThemeName, ThemeMode } from '.prisma/api-journeys-client'
+import { PrismaClient } from '.prisma/api-journeys-client'
 import { noop } from 'lodash'
```
Comment (tataihono) [inline apps/api-journeys/prisma/seed.ts:2]: is this still needed?

File: apps/api-journeys/prisma/factories/nuaEp8.ts [hunk 26]
```diff
@@ -0,0 +1,670 @@
...
+    })
+  }
+  await prisma.response.deleteMany({
+    where: { block: { journeyId: journey.id } }
+  })
+  await prisma.block.deleteMany({ where: { journeyId: journey.id } })
+  const nextBlockId = uuidv4()
+
+  //   first step
+  const step = await prisma.block.create({
```
Comment (tataihono) [inline apps/api-journeys/prisma/factories/nuaEp8.ts:30]: can you call this step1? the following steps your comment says // second step but the variable is step1 which is confusing.

**Review by tataihono — DISMISSED** [review]

**Review by csiyang — APPROVED** [review]

**Pull request merged**